libs/blocks/merch/merch.js

```diff
@@ -521,13 +521,21 @@ export async function initService(force = false, attributes = {}) {
     fetchEntitlements.promise = undefined;
     fetchCheckoutLinkConfigs.promise = undefined;
   }
-  const { commerce, locale: miloLocale } = getConfig();
+  const { commerce, env, locale: miloLocale } = getConfig();
```

Comment:
how about `env: miloEnv` ?
to stress out its not a commerce related env value but host env value. + works well with miloLocale we already have